libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions/Microsoft.IfCondition.schema

```diff
@@ -1,8 +1,8 @@
 {
     "$schema": "https://raw.githubusercontent.com/microsoft/botbuilder-dotnet/4.Future/schemas/component.schema",
     "$role": "unionType(Microsoft.IDialog)",
-    "title": "If Condition Action",
-    "description": "Action which conditionally decides which action to execute next.",
+    "title": "If-else branch",
```

Comment:
>If-else branch [](start = 14, length = 14)

Uggh....what's wrong with If Condition?

Reply:
We use condition for the overarching term: choose a condition - branching, loops etc are all actions that depend on a condition. See principle 1 below too (-: